packages/mdc-top-app-bar/foundation.js

```diff
@@ -43,11 +48,15 @@ class MDCTopAppBarFoundation extends MDCFoundation {
       hasClass: (/* className: string */) => {},
       addClass: (/* className: string */) => {},
       removeClass: (/* className: string */) => {},
+      setStyle: (/* attribute: string, value: string */) => {},
```

Comment:
Nit: Rename `attribute` to `property`

(See https://css-tricks.com/css-ruleset-terminology/)